### ISSUE TITLE
Is there any way that I can get some specific nodes of the AST

### ISSUE THREAD

**Issue by MajorGrinch** [eliben/pycparser#316]
Hi, I'd like to know if it's feasible to get a specific node of the AST, like given a function name I can get that specific FuncCall node for further analysis instead of just print it. I've read the source code of NodeVisitor, seems that it's just going through the whole AST without returning anything.

**Comment by eliben** [comment]
Yes, it's possible. Take a look at the https://github.com/eliben/pycparser/tree/master/examples directory, specifically examples starting w/ `func_`.
